platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/container/ContainerTestEnvironment.java

```
@@ -25,7 +32,16 @@ public class ContainerTestEnvironment implements TestEnvironment {
      * Constructor for the {@link ContainerTestEnvironment} class.
      */
     public ContainerTestEnvironment() {
-        network = new ContainerNetwork(timeManager, transactionGenerator);
+        final Path rootOutputDirectory = Path.of("build", "container");
+        try {
+            if (Files.exists(rootOutputDirectory)) {
+                FileUtils.deleteDirectory(rootOutputDirectory);
+            }
+            Files.createDirectories(rootOutputDirectory);
+        } catch (final IOException ex) {
+            log.warn("Failed to delete directory: {}", rootOutputDirectory, ex);
```

I think we should not swallow this Exception but fail the test.